resources/js/visual-editor/blocks/group/group.css

```
@@ -0,0 +1,68 @@
+    width: 100%;
+    flex-direction: inherit;
+    flex: 1;
+}
+
+.wp-block-group.is-layout-flex.block-editor-block-list__block > .block-list-appender:only-child::after {
+    content: "";
+    display: flex;
+    flex: 1 0 40px;
+    pointer-events: none;
+    min-height: 38px;
+    border: 1px dashed currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)